spec/classroom_spec.rb

```diff
@@ -0,0 +1,26 @@
+require '../classroom'
+require '../student'
+
+describe Classroom do
+  before(:each) do
+    @classroom = Classroom.new('A')
+  end
+
+  it 'takes 1 parameter and returns a Classroom object' do
+    expect(@classroom).to be_an_instance_of Classroom
+  end
+
+  it 'should have a label' do
+    expect(@classroom.label).to eq('A')
+  end
+
+  it 'should have an empty students array' do
+    expect(@classroom.students).to eq([])
+  end
+
+  it 'should add a student to the students array' do
+    student = Student.new('1A', 18, name: 'John', parent_permission: true)
+    @classroom.add_student(student)
+    expect(@classroom.students).to eq([student])
+  end
+end
```

Comment:
- [ ] Please note that when I run `RSpec spec` I encounter the below error message
![Screenshot 2022-04-13 152358](https://user-images.githubusercontent.com/44624138/163280318-f714e084-5c9f-425a-bed3-971ad2a7a6e9.jpg)
I'd suggest using `required relative` instead of required for better result

![Screenshot 2022-04-13 152126](https://user-images.githubusercontent.com/44624138/163280624-5ed01039-0d4c-48aa-9c21-c9f82de46a04.jpg)🤔😉

Reply:
Thank you for your review :+1: , we have implemented the required changes :smile: